cmd/dump.go

```diff
@@ -76,3 +95,64 @@ func dumpInstalledObjectsCmd(cmd *cobra.Command, args []string) error {
 	cmd.Printf("Dumped %d installed objects for package %s to %s\n", n, packageName, outputPath)
 	return nil
 }
+
+func dumpAgentPoliciesCmdAction(cmd *cobra.Command, args []string) error {
+	packageName, err := cmd.Flags().GetString(cobraext.PackageFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.PackageFlagName)
+	}
+
+	agentPolicy, err := cmd.Flags().GetString(cobraext.AgentPolicyFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.AgentPolicyFlagName)
+	}
+
+	outputPath, err := cmd.Flags().GetString(cobraext.DumpOutputFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.DumpOutputFlagName)
+	}
+
+	tlsSkipVerify, _ := cmd.Flags().GetBool(cobraext.TLSSkipVerifyFlagName)
+
+	var clientOptions []kibana.ClientOption
+	if tlsSkipVerify {
+		clientOptions = append(clientOptions, kibana.TLSSkipVerify())
+	}
+	kibanaClient, err := kibana.NewClient(clientOptions...)
+	if err != nil {
+		return errors.Wrap(err, "failed to initialize Kibana client")
+	}
+
+	switch {
+	case agentPolicy != "":
```

Comment:
Nit. Should we add a case to log an error if both agent policy and package name are set?

Reply:
True, as it is now it does not make sense to have both set at the same time.
I'll add a new case for that.
Thanks!